src/eventDispatchers/touchEventHandlers/touchStart.js

```diff
@@ -96,17 +96,23 @@ export default function (evt) {
       coords,
       distanceFromHandle
     );
+    
+    firstToolWithMoveableHandles.handleSelectedCallback(evt, moveableHandle, dataWithMoveableHandle);
```

Comment:
Do we need a check to make sure this callback exists on the tool?

Reply:
Its a virtual member function, with a default implementation for annotation tools, so we are safe to just call it 👍 .